### PR TITLE
Update submitted shootouts to match what's on the site today

### PR DESCRIPTION
**Pull request by bradcray**
* Michael's nbody silently replaced mine, so nbody3 is updated to reflect that
* Damian's nbody is version 4
* Jade's binary trees is version 4

### CODE REVIEW

File: test/studies/shootout/submitted/nbody4.perfkeys [hunk 1]
```diff
@@ -0,0 +1 @@
+nbody3.perfkeys
```
Comment (jabraham17) [inline test/studies/shootout/submitted/nbody4.perfkeys:1]: I'm guessing you meant to copy past `nbody3.perfkeys` here?

File: test/studies/shootout/submitted/nbody4.good
Comment (jabraham17) [inline test/studies/shootout/submitted/nbody4.good:1]: same here, I am guessing you meant to paste `nbody3.good` here?

File: test/studies/shootout/submitted/binarytrees4.good
Comment (jabraham17) [inline test/studies/shootout/submitted/binarytrees4.good:1]: same comments as nbody

File: test/studies/shootout/submitted/binarytrees4.perfkeys
Comment (jabraham17) [inline test/studies/shootout/submitted/binarytrees4.perfkeys:1]: same comments as nbody

**Review by jabraham17 — APPROVED** [review]

File: test/studies/shootout/submitted/nbody4.perfkeys (hunk 1 above)
Comment (bradcray) [reply test/studies/shootout/submitted/nbody4.perfkeys:1]: Huh… these were symbolic links, which I thought were OK to commit to git…  Is this just how GitHub renders them?  If you grab a copy of the branch, do you get the links out?

Comment (bradcray) [reply test/studies/shootout/submitted/nbody4.perfkeys:1]: Ah, I think the rendered contents of a symbolic link _is_ the path that the link is to?  So presumably it's the meta-information that makes this file "work right" when checked out.

Comment (jabraham17) [reply test/studies/shootout/submitted/nbody4.perfkeys:1]: Oh I see, in that case never mind, everything looks good!

**Pull request merged**